charts/redpanda/values.yaml

```diff
@@ -77,65 +82,66 @@ console:
     users:
     - name: admin
       password: change-me
       mechanism: SCRAM-SHA-512
 
 # TLS configuration
+# See https://docs.redpanda.com/docs/manage/kubernetes/security/kubernetes-tls/
 tls:
-  # Enable global TLS, which turns on TLS by default for all listeners
-  # Each listener must include a certificate name in its TLS section
-  # Any certificates in auth.tls.certs will still be loaded if enabled is false
-  # This is because listeners may enable TLS individually (see listeners.<listener name>.tls.enabled)
+  # Enable TLS globally for all listeners.
+  # Each listener must include a certificate name in its <listener>.tls object.
+  # To allow you to enable TLS for individual listeners, 
+  # certificates in auth.tls.certs are always loaded, even if tls.enabled is false.
+  # See listeners.<listener-name>.tls.enabled.
   enabled: false
-  # list all certificates below, then reference a certificate's name in each listener (see listeners.<listener name>.tls.cert)
+  # List all Certificates here.
   certs:
-    # This is the certificate name that is used to associate the certificate with a listener
-    # See listeners.<listener group>.tls.cert for more information
+    # This is the Certificate name.
+    # To apply the Certificate to a specific listener, 
+    # reference a Certificate's name in listeners.<listener-name>.tls.cert.
     default:
-      # Define an issuerRef to use your own custom pre-installed Issuer
+      # To use a custom pre-installed Issuer, add its name and kind here.
       # issuerRef:
       #   name: redpanda-default-root-issuer
       #   kind: Issuer   # Can be Issuer or ClusterIssuer
-      # The caEnabled flag determines whether the ca.crt file is included in the TLS mount path on each Redpanda pod
+      # The caEnabled flag determines whether the ca.crt file is included in the TLS mount path on each Pod.
       caEnabled: true
       # duration: 43800h
 
-#
 # External access configuration
+# See https://docs.redpanda.com/docs/manage/kubernetes/networking/networking-and-connectivity/
 external:
-  # Default external access value for each service
-  # Services can toggle external access per listener (ie. `listeners.<service name>.external.<listener name>.enabled`)
+  # Default external access value for each Service.
+  # You can toggle external access for each listener in listeners.<service name>.external.<listener-name>.enabled.
   enabled: true
-  # External access type. `NodePort` is the only acceptable value if defined.
-  # If undefined, then advertised listeners will be configured in Redpanda, but the helm chart will not create a Service. You must create a Service manually.
+  # Use `NodePort` or an empty string.
+  # If you use an empty string, advertised listeners are configured in Redpanda using external.addresses, 
+  # but the Helm chart does not create a Service. You must create a Service manually.
```

Comment:
Is this tested? I'm not sure it's allowed to be empty.

Reply:
Introduced in https://github.com/redpanda-data/helm-charts/pull/254 - I have tested it before and it worked